frontend/src/components/pages/AccessControl/index.tsx

```diff
@@ -202,11 +282,24 @@ const AccessControlPage = (): JSX.Element => {
                 />
               </Td>
               <Td>
-                <IconButton
-                  aria-label="Mark as active button"
-                  icon={<FaEllipsisV />}
-                  variant=""
-                />
+                <Popover>
+                  <PopoverTrigger>
+                    <IconButton
+                      aria-label="Mark as active button"
+                      icon={<FaEllipsisV />}
+                      variant=""
+                    />
+                  </PopoverTrigger>
+                  <PopoverContent width="inherit">
+                    <PopoverBody
+                      as={Button}
+                      bg="background.white.100"
+                      onClick={(e) => handleStatusChangePopoverTrigger(user)}
+                    >
+                      Mark as {user.active ? Filter.INACTIVE : Filter.ACTIVE}
```

Comment:
nit: can we define separate consts/enums for lowercase active/inactive strings shown here and in the modal? so we can show "Mark as active" instead of "Mark as Active"